mojaloop/values.yaml

```diff
@@ -2474,7 +2474,7 @@ ml-api-adapter:
     replicaCount: 1
     image:
       repository: mojaloop/ml-api-adapter
-      tag: v10.1.1
+      tag: v10.2.0
```

Comment:
Can you please update the tag in `ml-api-adapter-service:` as well to v10.2.0 (to keep it uniform and not run two different versions for both these services ?)